src/ModelContextProtocol.Core/Client/StdioClientTransport.cs

```diff
@@ -156,20 +156,21 @@ public async Task<ITransport> ConnectAsync(CancellationToken cancellationToken =
             // up the encoding from Console.InputEncoding. As such, when not targeting .NET Core,
             // we temporarily change Console.InputEncoding to no-BOM UTF-8 around the Process.Start
             // call, to ensure it picks up the correct encoding.
-#if NET
-            processStarted = process.Start();
-#else
```

Comment:
We should keep the simple `#if NET` (.NET Core) case which never sets Console.InputEncoding, so it cannot be the cause of your issues.